licenses-binary/LICENSE-protobuf.txt

```diff
@@ -1,42 +0,0 @@
-This license applies to all parts of Protocol Buffers except the following:
```

Comment:
If the binary artifact has the protobuf artifacts, we need to keep this license file.

Reply:
Got it, we will add it back

Reply:
Hi @WillemJiang ,we have addressed the issues in https://github.com/apache/incubator-celeborn/pull/1171/files, please have a check, thanks!